charts/azure/azure-disk-loss/rbac.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: azure-disk-loss-sa
+  namespace: default
+  labels:
+    name: azure-disk-loss-sa
+    app.kubernetes.io/part-of: litmus
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: azure-disk-loss-sa
+  namespace: default
+  labels:
+    name: azure-disk-loss-sa
+    app.kubernetes.io/part-of: litmus
+rules:
+  - apiGroups: 
+      - "" 
+      - "batch" 
+      - "apps" 
+      - "litmuschaos.io"
+    resources: 
+      - "jobs" 
+      - "pods" 
+      - "pods/log" 
+      - "events" 
+      - "deployments" 
+      - "replicasets" 
```

Comment:
Can we add rbac in same format as https://github.com/avaakash/chaos-charts/blob/azure-disk-loss/charts/azure/azure-instance-terminate/rbac.yaml and check for the minimal permissions